lib/global-styles.php

```diff
@@ -559,8 +542,6 @@ function gutenberg_experimental_global_styles_register_cpt() {
 	register_post_type( 'wp_global_styles', $args );
 }
 
-if ( gutenberg_is_experiment_enabled( 'gutenberg-full-site-editing' ) ) {
```

Comment:
Should we just replace the wrapping with gutenberg_experimental_global_styles_has_theme_json_support instead?
Would that be different?

Reply:
I wish we could, but the function we use to locate the file (`locate_template`) requires some of the WordPress constants to be loaded so it knows where to look for the file.